packages/svelte-vega/src/VegaEmbed.svelte

```diff
@@ -121,7 +121,7 @@
     }
   }
 
-  function handleError(error: Error) {
+  function handleError(error: Error): undefined {
```

Comment:
Why do we even return something? Shouldn't this function just return void?

Reply:
yeah wasn't sure if the function that called it depended on it returning undefined, don't think it does though